Auth/README.md

```diff
@@ -109,14 +117,21 @@ NSArray<id<FUIAuthProvider>> *providers = @[
   [[FUIEmailAuth alloc] init],
   [[FUIGoogleAuth alloc] init],
   [[FUIFacebookAuth alloc] init],
-  [[FUIPhoneAuth alloc] initWithAuthUI:[FUIAuth defaultAuthUI]]
+  [[FUIPhoneAuth alloc] initWithAuthUI:[FUIAuth defaultAuthUI]],
+  [FUIOAuth appleAuthProvider],
+  [FUIOAuth twitterAuthProvider],
+  [FUIOAuth githubAuthProvider],
+  [FUIOAuth microsoftAuthProvider],
+  [FUIOAuth yahooAuthProvider]
 ];
-_authUI.providers = providers;
+self.authUI.providers = providers;
 ```
 
 For Google Sign-in support, add custom URL schemes to your Xcode project
 (step 1 of the [implement Google Sign-In documentation](https://developers.google.com/firebase/docs/auth/ios/google-signin#2_implement_google_sign-in)).
 
+For Sign In with Apple support, add the Sign In with Apple capability to your entitlements file.
```

Comment:
Use italics?

Reply:
This formatting matches the Google Sign-in and Facebook login formatting above and below. Sign in with Apple is not usually italicized in Apple's documentation either ([example](https://support.apple.com/en-us/HT210318)).